app/helpers/project_settings_helper.rb

```diff
@@ -40,7 +40,7 @@ def project_settings_tabs
       {
         name: 'modules',
         action: { controller: '/project_settings/modules', action: 'show' },
-        label: :label_enabled_project_modules
+        label: :label_module_plural
```

Comment:
The menu item is then named differently from the heading of the page opened by it which, if I remember correctly, was an inconsistency remarked upon by @lindenthal, and which I would also like to avoid.